cds/container/concurrent_hopscotch_hash_set.h

```diff
@@ -0,0 +1,219 @@
+/*
+* concurrent_hopscotch_hash_set.h
+*
+*  Created on: 23 ���. 2017 �.
```

Comment:
You should change the authors to whom you copied this code from. This code bears a **striking** resemblance replete with bugs and deficiencies to [this](http://www.cs.tau.ac.il/~multi/2015a/chhm_files/Source_Code/CPP/fine%20grained/hopscotch.hpp) and [this](http://www.cs.tau.ac.il/~multi/2015a/chhm_files/Source_Code/CPP/fine%20grained/hopscotch.cpp). The header **clearly** states that this code is free to use so long as due acknowledgement is given.

You even have the _same comments_, the same dependencies to pthread, same variable names, other oddities like taking the values by pointer, and you also never use the value in the key/value hashmap.

Correct me if I'm wrong but this is similar to of extreme suspicion.